src/data/nat/choose/sum.lean

```diff
@@ -97,3 +97,43 @@ begin
 end
 
 end nat
+
+theorem int.alternating_sum_range_choose {n : ℕ} :
```

Comment:
Does it make sense to have a version that assume `n \ne 0` to avoid the `if` on the RHS?

Reply:
Yes, totally.